### PR TITLE
Perform baseline healthcheck

### PR DESCRIPTION
**Pull request by rcritten**
Run healthcheck on a default installation and ensure that there
are no failures. This test ensures that a fresh IPA installation
will pass healthcheck.

https://bugzilla.redhat.com/show_bug.cgi?id=1774032

### CODE REVIEW

**Comment by ssidhaye** [comment]
@rcritten Thanks for the test. Should we check if the output contains
Missing tracking for cert-database or Unknown certmonger id.
Otherwise the patch looks good.

**Comment by rcritten** [comment]
I don't think we need to explicitly check for this specific error. This baseline test will ensure that no errors at all are returned on a default installation which also covers the specific problem raised in the BZ.

**Comment by flo-renaud** [comment]
Hi @rcritten 
tests are green, please remove the temp commit.

**Comment by rcritten** [comment]
This doesn't require a rebase but PR https://github.com/freeipa/freeipa/pull/3645 added a new way to execute the test so I'm going to migrate to that method. I need to get PR https://github.com/freeipa/freeipa/pull/4317 merged first.

Removing ack for now.

**Comment by rcritten** [comment]
Tests passing after much flag issues. Adding back ack and pushing.

**Comment by rcritten** [comment]
Hang on, probably worth re-review since I rebased and made another slight change, adding back needs_review.

File: ipatests/test_integration/test_ipahealthcheck.py [hunk 1]
```diff
@@ -387,6 +392,20 @@ def test_source_ipa_roles_check_crlmanager(self):
             assert check["result"] == "SUCCESS"
             assert check["kw"]["key"] == "crl_manager"
             assert check["kw"]["crlgen_enabled"] is False
+        self.master.run_command(["ipa-crlgen-manage", "enable"])
```
Comment (flo-renaud) [inline ipatests/test_integration/test_ipahealthcheck.py:395]: I would put this command in a `finally:` block to make sure it is called even if the test fails.

**Comment by rcritten** [comment]
I implemented it as a fixture

**Comment by flo-renaud** [comment]
Hi @rcritten 
thanks for the updated patch, LGTM and tests are green. Please remove the temp commit and I will ack.

**Comment by rcritten** [comment]
master:

* 3022bb5fd200ea3b22fb1600401e95aaee2006ea Perform baseline healthcheck

**Pull request closed without merging**